smarty/cli_smarty.py

```diff
@@ -32,6 +32,14 @@ def main():
     """
     version_string = "%prog %__version__"
     parser = OptionParser(usage=usage_string, version=version_string)
+    
+    parser.add_option("-y", "--samplertype", metavar='SAMPLER_TYPE',
+                      action="store", type="string", dest='sampler_type', default='original',
+                      help="Choose between original or elemental sampler search (default = original).")
+                      
+    parser.add_option("-e", "--element", metavar='ELEMENT',
+                      action="store", type="string", dest='element', default=None,
+                      help="If you choose sampler elemental, you have to choose one element to search (number format, such as Oxygen = 8, Carbon = 6 and so on).")
```

Comment:
"one element to search (using the atomic number, i.e. --element=8 for oxygen or --elment=6 for carbon, etc.)"